dev/automated_tests/flutter_test/exception_handling_expectation.txt

```diff
@@ -47,8 +47,6 @@ When the exception was thrown, this was the stack:
 #[0-9]+ +.+ \(package:flutter_test[/\\]src[/\\]binding.dart:[0-9]+:[0-9]+\)
 #[0-9]+ +.+ \(package:flutter_test[/\\]src[/\\]binding.dart:[0-9]+:[0-9]+\)
 #[0-9]+ +.+ \(package:flutter_test[/\\]src[/\\]binding.dart:[0-9]+:[0-9]+\)
-#[0-9]+ +.+ \(package:flutter_test[/\\]src[/\\]binding.dart:[0-9]+:[0-9]+\)
-#[0-9]+ +.+ \(package:flutter_test[/\\]src[/\\]binding.dart:[0-9]+:[0-9]+\)
```

Comment:
AFAICT only one of these lines should be deleted.  The current Dart SDK should generate a trace with four stack frames associated with `binding.dart`

Reply:
The test fails if I only remove one line.

Reply:
Er, spoke too soon, I added a line to the wrong spot.

I'm going to just change this to assert we get at least one frame from this file though.